trieste/ask_tell_optimization.py

```diff
@@ -294,6 +294,13 @@ def __init__(
                 model.update(dataset)
                 model.optimize(dataset)
 
+            # summary_writer = logging.get_tensorboard_writer()
+            # if summary_writer:
+            #     with summary_writer.as_default(step=-1):
+            #         for tag, model in self._models.items():
+            #             with tf.name_scope(f"{tag}.model"):
+            #                 model.log()
+
```

Comment:
@uri-granta we forgot to log model at initial fit, this code is not working, what would be the best way to do this?